data/crac-creation/crac-creator-csa-profiles/src/main/java/com/powsybl/open_rao/data/crac_creation/creator/csa_profile/crac_creator/CsaProfileConstants.java

```diff
@@ -335,18 +341,18 @@ public enum HeaderType {
-        ASSESSED_ELEMENT_WITH_CONTINGENCY("assessedElementWithContingencyOverriding", "assessedElementWithContingency", "normalEnabled", "enabled", HeaderType.START_END_DATE),
-        ASSESSED_ELEMENT_WITH_REMEDIAL_ACTION("assessedElementWithRemedialActionOverriding", "assessedElementWithRemedialAction", "normalEnabled", "enabled", HeaderType.START_END_DATE),
-        CONTINGENCY_WITH_REMEDIAL_ACTION("contingencyWithRemedialActionOverriding", "contingencyWithRemedialAction", "normalEnabled", "enabled", HeaderType.START_END_DATE),
-        REMEDIAL_ACTION("remedialActionOverriding", "remedialAction", "normalAvailable", "available", HeaderType.START_END_DATE),
-        GRID_STATE_ALTERATION("gridStateAlterationOverriding", "gridStateAlteration", "normalEnabled", "enabled", HeaderType.START_END_DATE),
-        RANGE_CONSTRAINT("rangeConstraintOverriding", "rangeConstraint", "normalValue", "value", HeaderType.START_END_DATE),
-        REMEDIAL_ACTION_SCHEME("remedialActionSchemeOverriding", "remedialActionScheme", "normalArmed", "armed", HeaderType.START_END_DATE),
-        VOLTAGE_ANGLE_LIMIT("voltageAngleLimitOverriding", "voltageAngleLimit", "normalValue", "value", HeaderType.START_END_DATE),
-        CURRENT_LIMIT("currentLimitOverriding", "currentLimit", "normalValue", "value", HeaderType.SCENARIO_TIME),
-        VOLTAGE_LIMIT("voltageLimitOverriding", "voltageLimit", "normalValue", "value", HeaderType.SCENARIO_TIME);
+        CONTINGENCY("contingencyOverriding", REQUEST_CONTINGENCY, REQUEST_CONTINGENCIES_NORMAL_MUST_STUDY, REQUEST_CONTINGENCIES_OVERRIDE_MUST_STUDY, HeaderType.START_END_DATE),
+        ASSESSED_ELEMENT("assessedElementOverriding", REQUEST_ASSESSED_ELEMENT, NORMAL_ENABLED, OVERRIDE_ENABLED, HeaderType.START_END_DATE),
+        ASSESSED_ELEMENT_WITH_CONTINGENCY("assessedElementWithContingencyOverriding", REQUEST_ASSESSED_ELEMENT_WITH_CONTINGENCY, NORMAL_ENABLED, OVERRIDE_ENABLED, HeaderType.START_END_DATE),
+        ASSESSED_ELEMENT_WITH_REMEDIAL_ACTION("assessedElementWithRemedialActionOverriding", REQUEST_ASSESSED_ELEMENT_WITH_REMEDIAL_ACTION, NORMAL_ENABLED, OVERRIDE_ENABLED, HeaderType.START_END_DATE),
+        CONTINGENCY_WITH_REMEDIAL_ACTION("contingencyWithRemedialActionOverriding", REQUEST_CONTINGENCY_WITH_REMEDIAL_ACTION, NORMAL_ENABLED, OVERRIDE_ENABLED, HeaderType.START_END_DATE),
+        REMEDIAL_ACTION("remedialActionOverriding", REQUEST_REMEDIAL_ACTION, NORMAL_AVAILABLE, OVERRIDE_AVAILABLE, HeaderType.START_END_DATE),
+        GRID_STATE_ALTERATION("gridStateAlterationOverriding", "gridStateAlteration", NORMAL_ENABLED, OVERRIDE_ENABLED, HeaderType.START_END_DATE),
+        RANGE_CONSTRAINT("rangeConstraintOverriding", "rangeConstraint", NORMAL_VALUE, OVERRIDE_VALUE, HeaderType.START_END_DATE),
+        REMEDIAL_ACTION_SCHEME("remedialActionSchemeOverriding", CsaProfileConstants.REMEDIAL_ACTION_SCHEME, NORMAL_ARMED, OVERRIDE_ARMED, HeaderType.START_END_DATE),
```

Comment:
sadly not because we're defining the enum value REMEDIAL_ACTION_SCHEME which is "closer" than the class' constant
Maybe I can rename the constant to REQUEST_REMEDIAL_ACTION_SCHEME if you'd like.

Reply:
it's ok